modules/apps/transfer/keeper/relay.go

```diff
@@ -275,7 +275,7 @@ func (k Keeper) OnRecvPacket(ctx sdk.Context, packet channeltypes.Packet, data t
 
 		ctx.EventManager().EmitEvent(
 			sdk.NewEvent(
-				types.EventTypeDenomTrace,
+				types.EventTypeDenom,
 				sdk.NewAttribute(types.AttributeKeyTraceHash, token.Denom.Hash().String()),
 				sdk.NewAttribute(types.AttributeKeyDenom, voucherDenom),
```

Comment:
probably should update this to be `token.Denom` instead of `voucherDenom`, otherwise it's redundant